docs/osf-trolling-list/build_process.md

```diff
@@ -0,0 +1,127 @@
+`[OSFI001]` Build process
+-------------------------
+
+# `[OSFI0010]` There is no healthy OSF without reproducible builds
+
+## What are the reproducible builds?
+
+[Reproducible Builds](https://reproducible-builds.org/) and lot of its aspects
+is well explained on linked website.
+
+## Why those are in particular important for OSF?
+
+This is because knowing hashes of firmware components we can calculate expected
+TPM PCRs values. Knowing good PCR values and being able to predict further
+values after firmware update, we gaining following advantages:
```

Comment:
**By** knowing good PCR values and being able to predict further
values after firmware update, we **are** gaining **the** following advantages: ...

Reply:
@mikebdp2 please do as well use the `Insert a suggestion` feature for such changes.

Reply:
I pushed grammarly fixes for build process.